M/MAGEMin/build_tarballs.jl

```diff
@@ -56,25 +43,14 @@ install_license LICENSE
 
 augment_platform_block = """
     using Base.BinaryPlatforms
-    $(MPI.augment)
-    augment_platform!(platform::Platform) = augment_mpi!(platform)
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = supported_platforms(; exclude=p->Sys.isfreebsd(p) && arch(p) == "aarch64")
```

Comment:
If you're happy to use NLopt 2.9 or 2.10 you have also aarch64-freebsd and riscv64-linux builds:
```suggestion
platforms = supported_platforms()
```

Reply:
Ah yes, I forgot about NLopt. I was fixing to 2.8.0 because the newer version were yielding different results for different platform. I still need to open an issue for this. For now I am happy to still fix NLopt to 2.8.0, but will remove the constraint in the future